site/src/pages/components/accordion.explainer.mdx

```diff
@@ -73,6 +74,27 @@ Some examples of exclusive accordions on live sites:
 * [Parsons Table product page](https://www.roomandboard.com/catalog/dining-and-kitchen/tables/parsons-tables) at Room and Board (Style/Size/Top/Base accordion on right side of page)
 * ...
 
+# User Needs
+
+Developers regularly use exclusive accordions on the Web.
+Today, that generally means that they build their own widget
+rather than using a widget provided by the platform.
```

Comment:
... "because there is no such widget" or something? As worded, I'm left with the impression that the platform has something, but people don't use it.